modules/operators/migrations/20_0_0-beta_1-tap-response/index.spec.ts

```diff
@@ -0,0 +1,142 @@
+import * as path from 'path';
+import {
+  SchematicTestRunner,
+  UnitTestTree,
+} from '@angular-devkit/schematics/testing';
+import { createWorkspace } from '@ngrx/schematics-core/testing';
+import { tags } from '@angular-devkit/core';
+import { visitCallExpression } from '@ngrx/schematics-core/utility/visitors';
+import * as ts from 'typescript';
+import * as prettier from 'prettier';
+
+describe('migrate tapResponse', () => {
+  const collectionPath = path.join(__dirname, '../migration.json');
+  const schematicRunner = new SchematicTestRunner('schematics', collectionPath);
+  let appTree: UnitTestTree;
+
+  beforeEach(async () => {
+    appTree = await createWorkspace(schematicRunner, appTree);
+  });
+
+  const verifySchematic = async (input: string, output: string) => {
+    appTree.create('main.ts', input);
+
+    const tree = await schematicRunner.runSchematic(
+      '20.0.0-beta_1-tap-response',
+      {},
+      appTree
+    );
+
+    const actual = prettier.format(tree.readContent('main.ts'), {
+      parser: 'typescript',
+    });
+
+    expect(actual).toBe(
+      prettier.format(output, {
+        parser: 'typescript',
+      })
+    );
+  };
+
+  it('should migrate basic tapResponse signature', async () => {
```

Comment:
We prefer not to use the "should" form.

```suggestion
  it('migrates basic tapResponse signature', async () => {
```